### DISCUSSION TITLE
Question about Control Surfaces for Razer Tartarus Pro

### DISCUSSION THREAD

**Discussion by RAZER-USER** [CommandPost/CommandPost#3258] [Technical Support] [answered]
Hi!!

Working on a Mac in the Control Surfaces Razer interface within the Selected Control window where button assignment takes place, is it possible to map the keys 'SHIFT,' 'CMD' or 'OPT' by themselves? For example I only see "SHIFT and .." another key when I make the attempt and scroll through the choices. Looking for a solution that will allow me to isolate those commands on the Razer keypad. I didn't see anything like this when I searched for similar discussions but I'm new and hope that I'm using the search correctly!

Thanks in advance!!

**Comment by latenitefilms** [comment]
To use control surface keys as regular modifier keys, you need both **press** and **release** actions - for example:

<img width="1280" alt="image" src="https://github.com/CommandPost/CommandPost/assets/22286696/e23436c0-ec23-4bbd-a684-9cee44ff2502">

<img width="1280" alt="image" src="https://github.com/CommandPost/CommandPost/assets/22286696/0a15041a-b1d3-488a-9e47-6733ad9a5ab9">

**Reply by RAZER-USER** [reply, marked as answer]
Got it!! This is a massive step forward for me and I imagine many Razer users on Mac. Can't thank you enough for creating this. Will spread the good news
